css/login.css

```diff
@@ -0,0 +1,164 @@
+body{
+	background-color: #555555;
+	margin: 0;
+	font-family: 'Roboto', sans-serif;
+	min-width: 1200px;
+}
+
+.rectangle{
+	width: 100%;
+	height: 640px;
+}
+
+.firstRectangle{
+	background-image: url('../img/feed.jpg');
+	background-blend-mode: overlay;
+}
+
+.whiteRectangle{
+	width: inherit;
+	height: inherit;
+	background-color: #CBF6FF;
+	opacity: 0.3;
+	position: absolute;
+}
+
+.logInBox{
+	width: 375px;
+	height: 400px;
+	margin-left: 400px;
+	background-color: #F5F5F5;
+	opacity: 1!important;
```

Comment:
Why is this so `!important`? As a best practice, any time you feel the need to use `!important`, stop and think if there are other ways around it. If there are truly no other ways of achieving what you want, then put in a comment explaining why the importance. Saves you from a helluvalotta headaches in the long run, I promise!